.github/workflows/continuous-integration.yml

```diff
@@ -89,6 +89,11 @@ jobs:
           pip install yamllint
           python .github/scripts/ci/run_yamllint.py --config .yamllint.yaml
 
+      - name: find broken wikilinks
+        run: |
+          ARTICLES=$( git diff --diff-filter=d --name-only ${{ github.sha }}^ ${{ github.sha }} "wiki/**/*.md" "news/*.md" )
+          python .github/scripts/ci/find_broken_wikilinks.py $ARTICLES
```

Comment:
FYI it also detects regular broken links, so I'd just call it `find_broken_links.py`

Reply:
it's not intended for regular links, and even skips any that it thinks are broken links. we can't check for regular links apart from format issues unless we try ping that link

Reply:
I know it's not a universal liveness checker. what I wanted to say was that it caught https://github.com/ppy/osu-wiki/pull/6641/files#diff-c5cdbe755d54c6909a866faa59684ae07e9be41c799e0675f7de9e4efa66bcbe